audio-worklet/lib/param-controller.js

```diff
@@ -35,7 +35,17 @@ class ParamController {
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
-    this.controller_.max = options.max || 0;
+
+    if (options.max == null)
+      this.controller_.max = 1;
+    else
+      this.controller_.max = options.max;
```

Comment:
I learned this one the hard way. Suppose you want the maximum possible value to be 0 (as is the case with the noise gate threshold for example.). Since 0 evaluates to false, it will take on whatever value was supplied by the second argument if that value is non-zero. So this method for setting default values only works when the second argument is zero.